javascript/javascript1/week2/Student-manager.js

```diff
@@ -0,0 +1,49 @@
+
+    const class07Students = [];
+    function addStudentToClass(studentName) {
+        let str='';
+        let number=getNumberOfStudents();
+        if(studentName===''){
```

Comment:
Use [trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim) to check long empty strings